server/service/src/sync/init_programs_data.rs

```diff
@@ -323,6 +322,26 @@ fn encounter_2() -> hiv_care_encounter::HivcareEncounter {
                 exam.blood_pressure = Some(125.0);
             },
         ));
+        e.events = Some(vec![
+            hiv_care_encounter::EncounterEvent {
+                datetime: time
+                    .checked_add_signed(Duration::weeks(1))
+                    .unwrap()
+                    .to_rfc3339(),
+                group: Some("HivCareEncounterDispensingStatus".to_string()),
+                type_: "status".to_string(),
+                name: Some("Interrupted".to_string()),
+            },
+            hiv_care_encounter::EncounterEvent {
+                datetime: time
+                    .checked_add_signed(Duration::weeks(2))
+                    .unwrap()
+                    .to_rfc3339(),
+                group: Some("HivCareEncounterDispensingStatus".to_string()),
+                type_: "status".to_string(),
+                name: Some("Lost".to_string()),
```

Comment:
should this be `Lost to follow up` to match the 'number of days dispensed' event?